### ISSUE TITLE
Alerts color contrast

### ISSUE THREAD

**Issue by lwrubel** [gwu-libraries/sfm-ui#666]
Check contrast on bootstrap alerts using Web AIM color contrast tool. Should be 4.5:1 or above. Adjust color globally.

**Comment by lwrubel** [comment]
Once colors determined, reassign to programmer to implement.

**Comment by yonahbg** [comment]
I went through and discovered that the green default button doesn't pass the test. The other buttons pass, but not by much. Thoughts?
![color guide draft-01](https://cloud.githubusercontent.com/assets/23175249/23075041/f7990920-f508-11e6-8a7a-ecaa364ad6f0.jpg)

**Comment by lwrubel** [comment]
@yonahbg were you able to find any color combinations that keep the color-on-color (e.g. green-on-green) scheme without going all the way to black and white lettering? I'm thinking of this in the alerts messages appearance in particular.

**Comment by yonahbg** [comment]
@lwrubel I found these. Much lower scores, but they pass the 4.5 threshold.
![color guide draft 2-02](https://cloud.githubusercontent.com/assets/23175249/23307653/481824cc-fa76-11e6-93b2-9935b199ad6f.jpg)

Also, I can get a pdf with the values copyable when we pick a color guide that works

**Comment by justinlittman** [comment]
Not going to fix at this time.
